src/client/utils/tracking/tracking.ts

```diff
@@ -44,6 +47,11 @@ export const getUtmParamsFromCookie = (): UtmParams | undefined => {
   }
 }
 
+export enum SeBundleTypes {
+  SeAccidentBundle = 'SE_ACCIDENT_BUNDLE',
+  SeAccidentBundleStudent = 'SE_ACCIDENT_BUNDLE_STUDENT',
+}
```

Comment:
I see that we both added this types @mehdielket :) Let's see if I can use yours, I'm not sure I need to know about brf, house or rent though.

Reply:
Yes. I confirmed this with Sonny and Fred so we need to granularity

Reply:
For sure, using your type instead 🙌